### ISSUE TITLE
Animation steps after 'repeat' never happen

### ISSUE THREAD

**Issue by dansisan** [mdgriffith/elm-style-animation#39]
It seems `repeat` prevents subsequent steps from animating?

```
Animation.interrupt
    [ Animation.to
        [ Animation.rotate (turn 0.5) ]
    -- steps before repeat work fine, as expected
    , ( Animation.repeat 3
        [ Animation.to
            [ Animation.translate (px 0) (px 150) ]
        , Animation.to
            [ Animation.translate (px 0) (px 0) ]
        ] )
    -- steps after repeat never happen
    , Animation.to
        [ Animation.rotate (turn 0.0) ]
    ]
```

**Comment by mdgriffith** [comment]
Thanks for the issue!  I found the problem and released a fixed as 3.5.5.

**Comment by dansisan** [comment]
Impressive turnaround! I confirmed 3.5.5 fixes the problem on my end. Thanks!
